eng/common/testproxy/test-proxy.ps1

```diff
@@ -1,162 +0,0 @@
-Set-StrictMode -Version 4
```

Comment:
The removal of this file will affect `test-proxy-standalone-tool.yml`. Just a heads up. We use this to install the appropriate proxy for the platform

Reply:
The yaml calls into `install-test-proxy.ps1` which has been updated to use the renamed/refactored file.

Reply:
Not directly but in `https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/testproxy/install-test-proxy.ps1`